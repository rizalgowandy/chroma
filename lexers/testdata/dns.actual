; /etc/bind/zones/example.org
$TTL 3h;
@	IN	SOA	ns.example.org. example.org. (
	1 ; serial
	3h; refresh timeout
	1h; retry timeout
	1w; expire timeout
	1h; negative caching TTL
)
@	IN	NS		ns.example.org.  ; name server
@	IN	MX	10	example.org.     ; mail exchange server

ns	IN	A		1.2.3.4
@	IN	A		1.2.3.4

ns	IN	AAAA	ef::1
@	IN	AAAA	ef::1

www	IN	CNAME	example.org.
