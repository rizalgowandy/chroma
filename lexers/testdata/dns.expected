[
  {"type":"Comment","value":"; /etc/bind/zones/example.org"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"NameAttribute","value":"$TTL"},
  {"type":"TextWhitespace","value":" "},
  {"type":"LiteralStringChar","value":"3h"},
  {"type":"Comment","value":";"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"NameClass","value":"@"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"IN"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"SOA"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"NameProperty","value":"ns.example.org. example.org. "},
  {"type":"Punctuation","value":"("},
  {"type":"TextWhitespace","value":"\n\t"},
  {"type":"LiteralStringChar","value":"1"},
  {"type":"TextWhitespace","value":" "},
  {"type":"Comment","value":"; serial"},
  {"type":"TextWhitespace","value":"\n\t"},
  {"type":"LiteralStringChar","value":"3h"},
  {"type":"Comment","value":"; refresh timeout"},
  {"type":"TextWhitespace","value":"\n\t"},
  {"type":"LiteralStringChar","value":"1h"},
  {"type":"Comment","value":"; retry timeout"},
  {"type":"TextWhitespace","value":"\n\t"},
  {"type":"LiteralStringChar","value":"1w"},
  {"type":"Comment","value":"; expire timeout"},
  {"type":"TextWhitespace","value":"\n\t"},
  {"type":"LiteralStringChar","value":"1h"},
  {"type":"Comment","value":"; negative caching TTL"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"Punctuation","value":")"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"NameClass","value":"@"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"IN"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"NS"},
  {"type":"TextWhitespace","value":"\t\t"},
  {"type":"NameProperty","value":"ns.example.org. "},
  {"type":"TextWhitespace","value":" "},
  {"type":"Comment","value":"; name server"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"NameClass","value":"@"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"IN"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"MX"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"LiteralStringChar","value":"10"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"NameProperty","value":"example.org. "},
  {"type":"TextWhitespace","value":"    "},
  {"type":"Comment","value":"; mail exchange server"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"NameClass","value":"ns"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"IN"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"A"},
  {"type":"TextWhitespace","value":"\t\t"},
  {"type":"LiteralNumberInteger","value":"1.2.3.4"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"NameClass","value":"@"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"IN"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"A"},
  {"type":"TextWhitespace","value":"\t\t"},
  {"type":"LiteralNumberInteger","value":"1.2.3.4"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"NameClass","value":"ns"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"IN"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"AAAA"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"LiteralNumberIntegerLong","value":"ef::1"},
  {"type":"TextWhitespace","value":"\n"},
  {"type":"NameClass","value":"@"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"IN"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"AAAA"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"LiteralNumberIntegerLong","value":"ef::1"},
  {"type":"TextWhitespace","value":"\n\n"},
  {"type":"NameClass","value":"www"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"IN"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"Keyword","value":"CNAME"},
  {"type":"TextWhitespace","value":"\t"},
  {"type":"NameProperty","value":"example.org."},
  {"type":"TextWhitespace","value":"\n"}
]
